cmake/FindLibClang.cmake

```diff
@@ -15,13 +15,16 @@ if( ${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
   set(TRIQS_LIBCLANG_CXX_ADDITIONAL_FLAGS "" CACHE STRING "Additional flags to be passed to libclang when parsing with clang")
 
   set(CLANG_COMPILER "clang++")
-  set(CLANG_OPT  -std=c++1y -stdlib=libc++ )
+  set(CLANG_OPT -stdlib=libc++ )
 
 else()
 
 SET(TRIAL_LIBRARY_PATHS
+ ENV LIBRARY_PATH
```

Comment:
No, since you have to get rid of the : and put spaces between the different folders in the LIBRARY_PATH.

This is done by cmake automatically, see signature here:

https://cmake.org/cmake/help/v3.0/command/find_library.html